src/coreclr/gc/gc.cpp

```diff
@@ -7151,7 +7151,6 @@ void gc_mechanisms::init_mechanisms()
     found_finalizers = FALSE;
 #ifdef BACKGROUND_GC
     background_p = gc_heap::background_running_p() != FALSE;
-    allocations_allowed = TRUE;
```

Comment:
So this business about allocations_allowed is just to remove test code that is no longer required?

Reply:
yes, that's all there is.